src/api/swagger.yaml

```diff
@@ -3019,6 +3027,13 @@ paths:
         - userAuth: [admin, read-only-admin, user]
       description: Get minimum host requirements.
       operationId: GetHostRequirements
+      parameters:
```

Comment:
We can enhance UI for that. Are SNO HW requirements in the UI tracked somewhere? @jelkosz

Reply:
https://issues.redhat.com/browse/MGMT-7135